test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -313,6 +314,13 @@ public void uncaughtException(Thread thread, Throwable throwable)
             else
                 logger.error("uncaught exception in thread {}", thread, throwable);
         }
+
+        @Override
+        public String toString()
+        {
+            IInvokableInstance delegate = this.delegate;
```

Comment:
did this in https://issues.apache.org/jira/browse/CASSANDRA-16213 as its useful for debugging.